MainModule/Server/Commands/Moderators.lua

```diff
@@ -4927,6 +4927,87 @@ return function(Vargs, env)
 				end
 			end
 		};
+		
+		CreateLeaderstats = {
+			Prefix = Settings.Prefix;
+			Commands = {"createstat", "cstat"};
+			Args = {"name"};
+			Description = "Creates a new leaderstat";
+			AdminLevel = "Moderators";
+			Function = function(plr: Player, args: {string})
+
+				local statName = assert(args[1], "Missing stat name")
+
+				for _, v in service.GetPlayers(plr, args[1]) do
+					local leaderstats = plr:FindFirstChild("leaderstats")
+
+					if leaderstats and (leaderstats:IsA("Folder") or leaderstats:IsA("StringValue")) then
+						local statName = Instance.new("IntValue")
```

Comment:
Prefer `service.New` over `Instance.new` in Adonis code:
```lua
service.New("IntValue", {
    Parent = leaderstats,
    Name = args[1],
    Value = 0
}
```

Reply:
Will do